Source/diablo.cpp

```diff
@@ -1766,12 +1772,12 @@ void InitKeymapActions()
 	    SpellBookKeyPressed,
 	    nullptr,
 	    CanPlayerTakeAction);
-	for (int i = 0; i < 4; ++i) {
+	for (size_t i = 0; i < QUICK_MESSAGE_OPTIONS; ++i) {
 		sgOptions.Keymapper.AddAction(
 		    "QuickMessage{}",
 		    N_("Quick Message {}"),
 		    N_("Use Quick Message in chat."),
-		    SDLK_F9 + i,
+		    SDLK_KP_0 + i,

```

Comment:
`SDLK_KP_0` comes after `SDLK_KP_9` in SDL2.

Reply:
I suppose I should also point out that our default keybinds are set up to closely match the vanilla's keybinds. So the correct default would be to map the first four quick messages to F9-F12 and then leave the rest of them unmapped.

Reply:
Oh right, what is interesting is I've found that SDL2 goes with SDLK_KP_0 after SDLK_KP_9, and you can see me applying this logic in `options.cpp`, then I've went back to this and it turns out it works either way for some reason, so I left it out as it is, but anyways - I've changed this.